gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #210: Can signup with an invalid date of birth

### Description

Add validations to check if date of birth is a date in the past in signup and settings forms.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1. Signup using date of birth as a date in future. It should throw an error "Value must be {today's date} or earlier"
2. Sign in as a registered user and change the date of birth to a date in futute. It should throw the same error "Value must be {today's date} or earlier"

### Miscellaneous Comments

![signup page](https://user-images.githubusercontent.com/38910831/70451092-cad87a00-1a72-11ea-9adc-e7169e258da4.PNG)
![settings page](https://user-images.githubusercontent.com/38910831/70451093-cb711080-1a72-11ea-8cd7-ccff812f3d65.PNG)

### Checklist
